assignments/arrays.js

```diff
@@ -75,25 +75,61 @@ for (let i = 0; i < inventory.length; i++){
 let lastCar = 0;
 console.log();
 
+for (let i = 0; i < inventory.length; i++){
+    if (inventory[i] === inventory[inventory.length -1]){
+        console.log(inventory[i]);
+    }
+}
+
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+
+
+for (let i = 0; i < inventory.length; i++){
+    let model = inventory[i].car_model;
+    carModels.push(model);
+    carModels.sort();
+}
+
+console.log(carModels);
+
+
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+
+for (let i = 0; i < inventory.length; i++){
+    year = inventory[i].car_year;
+    carYears.push(year);
+}
+
+console.log(carYears);
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
 let oldCars = [];
-console.log(); 
+
+for (let i = 0; i < carYears.length; i++){
+    if (carYears[i] < 2000){
```

Comment:
It's great that you used the existing `carYears` array that you populated on Challenge 4! ⭐️